### PR TITLE
Document UUID support for authors

### PR DESCRIPTION
**Pull request by matic0basle**
Thanks to Elliott_ to helping me with this.

### CODE REVIEW

**Comment by matic0basle** [comment]
In the update I just made I changed the example and removed the note that was also removed on the thread due to the edit from last

https://oc.tc/forums/topics/54d720c2c29635cadd000bf3

File: src/modules/main.haml [hunk 1]
```diff
@@ -161,17 +161,29 @@
...
+                                    %code uuid
+                                %td UUID used to define a player.
+                                %td
+                                    %span.label.label-primary String
+
+                %br
+                A players UUID is a unique code to each player that is used to keep track of each player even if they change their name. You can check player UUID's on
+                = succeed "." do
+                    %a{:href => "http://mcuuid.net/"} mcuuid.net
+                %br
```
Comment (McSpider) [inline src/modules/main.haml]: You should move this whole block (starting at the first `%br`) down into the markdown parser, removing the `succeed do` nonsense.

Also document the fact that if an author or contributor is defined without a uuid that player will not get any mapmaker perks on the map.

**Comment by matic0basle** [comment]
@McSpider I think that should be it.

**Comment by McSpider** [comment]
Merged: 06995e03f592376751ab97578bece42a88b9de85

Thanks!

**Pull request closed without merging**